docs/docs/signing.md

```diff
@@ -23,8 +23,9 @@ Using signatures to link binaries into a single upgrade path is the same techniq
 permissions and stored data from old to new apps.
 
 Signing is also used to authorise which enclaves can start. By default Intel chips won't start an enclave unless it's 
-signed by an Intel key, but this behaviour can be changed and on Azure (see ["Deploying to Azure"](azure.md)) enclaves
-can be self-signed. If you want to use older hardware getting whitelisted by Intel is free and can be done quickly. 
+signed by an Intel key, but this behaviour can be changed and on Azure (see ["Deploying to Azure"](machine-setup.md)),
+enclaves can be self-signed. If you want to use older hardware getting whitelisted by Intel is free and can be done 
```

Comment:
I would add a `,`, and add a `this` or `it` i.e. `if you want to use older hardware, getting this whitelisted by Intel`...

Reply:
Done.